packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -146,6 +143,13 @@ class MenuContent extends React.Component {
     }
   }
 
+  setPageHeight(node) {
```

Comment:
You shouldn't need to do this calculation. Popup should handle this without setting an explicit height on the content. You can change this https://github.com/cerner/terra-framework/pull/992/files#diff-8821ddb6189a1accf05f7aef26338373R344 to be 
`const contentHeight = this.props.isHeightBounded && '100%' ;
And update the content container's `fill` prop to be just `this.props.isHeightBounded`

Reply:
updated here b91205212bdd630b10eb4d5f5748cd63e33ac4a0